scripts/submit_form.js

```diff
@@ -0,0 +1,113 @@
+// Google Apps Script to submit a form to a Google Sheet and send an email notification
+
+var SCRIPT_PROPERTIES = PropertiesService.getScriptProperties();
+const EMAIL_ADDRESS = "";  // set this!
+
+const formIdToNameMap = new Map([
+  ["feedback", "Give Feedback"],
+  ["report_missing_info", "Report Missing or Outdated Information"],
+  ["partner_with_freefrom", "Partner with FreeFrom"],
+  ["build_collective_survivor_power", "Build Collective Survivor Power"],
+  ["policy_ideas", "Share your Policy Ideas"],
+]);
+
+function setup() {
+  var doc = SpreadsheetApp.getActiveSpreadsheet();
+  SCRIPT_PROPERTIES.setProperty("key", doc.getId());
+}
```

Comment:
This is a setup function that needs to be run once from the account deploying the app!